posthog/models/team/team.py

```diff
@@ -408,6 +408,13 @@ def revenue_analytics_config(self):
         config, _ = TeamRevenueAnalyticsConfig.objects.get_or_create(team=self)
         return config
 
+    @cached_property
+    def marketing_analytics_config(self):
+        from .team_marketing_analytics_config import TeamMarketingAnalyticsConfig
+
+        config, _ = TeamMarketingAnalyticsConfig.objects.get_or_create(team=self)
+        return config
```

Comment:
BTW, you might wanna backfill these eventually for older teams, this way you can almost-always assume they exist. Django is kinda bad for this kind of behavior, but this has been enough for me.